src/helm/benchmark/scenarios/legal_summarization_scenario.py

```diff
@@ -0,0 +1,191 @@
+from pathlib import Path
+
+from typing import List, Optional, Any
+
+import datasets
+from datasets import load_dataset
+
+from .scenario import Scenario, Instance, Reference, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+_ALL_LANGUAGES = {
+    "bulgarian": "bg",
+    "czech": "cs",
+    "dutch": "nl",
+    "estonian": "et",
+    "french": "fr",
+    "greek": "el",
+    "irish": "ga",
+    "latvian": "lv",
+    "maltese": "mt",
+    "portuguese": "pt",
+    "slovak": "sk",
+    "spanish": "es",
+    "croatian": "hr",
+    "danish": "da",
+    "english": "en",
+    "finnish": "fi",
+    "german": "de",
+    "hungarian": "hu",
+    "italian": "it",
+    "lithuanian": "lt",
+    "polish": "pl",
+    "romanian": "ro",
+    "slovenian": "sl",
+    "swedish": "sv",
+}
+
+# IMPORTANT: So far, we only support English, since we would need to adapt the metrics for scoring the other languages.
+_SUPPORTED_LANGUAGES = {
+    "english": "en",
+}
+
+
+def _clean_and_truncate(text: str, max_length: Optional[int] = None) -> str:
+    text = text.replace("\n", " ")
+    return " ".join(text.split()[:max_length])
+
+
+class LegalSummarizationScenario(Scenario):
```

Comment:
Is this code specific to legal at all, or is it just more summarization datasets?  Just wonder if we can unify all the summarization datasets?

Reply:
No, in principle it is very similar to the other summarization tasks. We were debating whether it makes sense to integrate them with the other summarization datasets. I see three possibilities:
1. together with the other summarization datasets in the same scenarios
2. all legal summarization scenarios together (this PR)
3. each summarization dataset as its own scenario

1. Has the advantage of more code reuse but on the other hand, it is harder to evaluate models on only legal datasets if they are merged with general summarization tasks.
3. leads to lots of boilerplate code